src/client/components/input/drag-and-drop-image.js

```diff
@@ -14,6 +29,11 @@ class DragAndDropImage extends React.Component {
 		};
 		ev.dataTransfer.setData('text', JSON.stringify(data));
 	}
+	/**
+	* Renders an image of the achievement badges which supports
```

Comment:
Better! How about `Renders an image of a particular achievement badge, which can be dragged to set the user's publicly showcased achievements`?